### PR TITLE
Copy updates to Palace invite form

### PR DESCRIPTION
**Pull request by TheDancingClown**
## 📝 A short description of the changes

* Copy updates to palace invite form

## 🔗 Link to the relevant story (or stories)

* https://app.asana.com/0/1200504523179343/1207204405967425

## :shipit: Deployment implications

* None

## ✅ Checklist

- [x] Features that cannot go live are behind a feature flag/env var or specify deploy date and open PR as draft 
- [x] I have checked that commit messages make sense and explain the reasoning for each change
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have squashed any unnecessary or part-finished commits

## 🖼️ Screenshots (if appropriate - no PII/Prod data):

Screenshot 2024-05-01 at 16 13 08
Screenshot 2024-05-02 at 08 59 32

### CODE REVIEW

**Review by matthewford — APPROVED** [review]

File: app/decorators/trigger_at_decorator.rb [hunk 1]
```diff
@@ -1,8 +1,12 @@
 module TriggerAtDecorator
-  def formatted_trigger_time(bold = true)
+  def formatted_trigger_time(bold = true, format=nil)
     return placeholder unless object.trigger_at
 
-    trigger_on = object.trigger_at.strftime("%-d %b %Y")
+    if format.present? && format == "full"
+      trigger_on = object.trigger_at.strftime("%-d %B %Y,")
```
Comment (dreamfall) [inline app/decorators/trigger_at_decorator.rb:6]: I don't think comma is intentional. 

```suggestion
      trigger_on = object.trigger_at.strftime("%-d %B %Y")
```

File: app/views/admin/form_answers/winners_components/_palace_attendee.html.slim [hunk 2]
```diff
@@ -35,9 +35,20 @@
...
+          br
+          label Dietary needs
+          p
+            - if pa.dietary_requirements.present?
+              = pa.dietary_requirements
+            - else
+              ' -
+          br
+          label Does the attendee require disabled access?
+          p= pa.disabled_access? && pa.disabled_access.presence || "-"
```
Comment (dreamfall) [inline app/views/admin/form_answers/winners_components/_palace_attendee.html.slim:47]: one check should be enough
```suggestion
          p= pa.disabled_access.presence || "-"
```

**Review by dreamfall — CHANGES_REQUESTED** [review]

File: app/decorators/trigger_at_decorator.rb (hunk 1 above)
Comment (TheDancingClown) [reply app/decorators/trigger_at_decorator.rb:6]: Also changing the month format from Jul to July

Comment (TheDancingClown) [reply app/decorators/trigger_at_decorator.rb]: Refactored trigger_at_decorator

**Review by dreamfall — APPROVED** [review]

**Pull request merged**